python/az/aro/azext_aro/custom.py

```diff
@@ -183,10 +184,32 @@ def aro_create(cmd,  # pylint: disable=too-many-locals
                 visibility=ingress_visibility or 'Public',
             )
         ],
+        service_principal_profile=None,
+        platform_workload_identity_profile=None,
     )
 
-    sp_obj_ids = [client_sp_id, rp_client_sp_id]
-    ensure_resource_permissions(cmd.cli_ctx, oc, True, sp_obj_ids)
+    if enable_managed_identity is True:
+        oc.platform_workload_identity_profile = openshiftcluster.PlatformWorkloadIdentityProfile(
+            platform_workload_identities=platform_workload_identities
+        )
+
+        oc.identity = openshiftcluster.Identity(
+            type='UserAssigned',
+            user_assigned_identities={mi_user_assigned: {}}
+        )
+
+        # TODO - add cluster identity here
```

Comment:
Re: the comment on line 201, haven't you already added the cluster identity on line 198?

Reply:
Yes. I likely re-introduced the comment with a bad rebase. I'll remove it.